src/Admin/Controllers/MiscController.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Admin.Controllers
+{
+    public class MiscController : Controller
```

Comment:
For Admin, please put into the `HomeController` vs. creating a new one for these. I know it seems inconsistent, however if there's already a home controller, that is the default route in ASP.NET which captures the `~/` root application path handlers such as these.

Reply:
That makes sense. It's been updated.